lib/src/signed_video_h26x_auth.c

```diff
@@ -319,6 +370,7 @@ verify_hashes_with_hash_list(signed_video_t *self, int *num_expected_nalus, int
         } else {
           item->validation_status = item->first_verification_not_authentic ? 'N' : '.';
           item->need_second_verification = false;
+          order_ok = true;
```

Comment:
This line has no impact since `order_ok` is always `true` when reaching this point.